server/default.nix

```diff
@@ -1,15 +1 @@
-{ # We expect getPkgs to be a lambda that gives us a package set. Example:
-  # the lambda in `nixpkgs/default.nix` without any arguments passed to it.
-  # That allows us to pass in config later, while allowing users to pass in
-  # their own version of Nixpkgs.
-  getPkgs ? import ../nix/nixpkgs.nix
-}:
-
-let
-  pkgs = getPkgs { };
-  icepeak-server = pkgs.haskellPackages.callPackage ./server.nix { };
-in
-  {
-    icepeak-server = icepeak-server;
-    stack = pkgs.haskellPackages.stack;
-  }
```

Comment:
We might be relying on this file in this form in other projects that pull in icepeak.

Would it be an idea to introduce the development environment as part of the top-level default.nix that we already have?